rllib/utils/backend.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import absolute_import
```

Comment:
Consider 'framework.py' for consistency.

Reply:
Ok, makes sense. Will rename.